File: C++ Projects/Programming Assignment 4/Programming Assignment 4/Viewer.cpp
// COSC 2320 Spring 2014 
// Name: Nguyen Doan
// Programming Assignment 4
// This is my own work; I will not post
#include "Viewer.h"
#include "Driver.h"

Viewer::Viewer(void)
{
}


Viewer::~Viewer(void)
{
}

void Choose()
{
	Driver drive;
	int input = 0;
	cout<<"Click 1 to start"<<endl;
	cout<<"Click 2 to exit"<<endl;
	cin>>input;

	if(input = 1)
	{
		drive.Open();
		drive.Encode();
		drive.Decode();
		drive.Print();
		system("PAUSE");
	}
	else if(input = 2)
	{
		system("PAUSE");
	}
}

File: Java/Programming Assignment 3/Programming Assignment 3 Java/src/programmingassignment3java/Viewer.java
// COSC 2320 Spring 2014
// Name: Nguyen Doan
// Programming Assignment 3
// This is my own work; I will not post

package programmingassignment3java;

import java.io.IOException;

public class Viewer
{
    void callMenu() throws IOException
    {
        ContactInterface intfac = new ContactInterface();
        intfac.Menu();
    }
}
